wordpress-mobile/WordPress-iOS
# Update outdated pods

Fixes #8195 

To test:
- Check that the tests pass.
- Check that the SVProgressHUD works properly.
- Check that you can Hide/Unhide a site.
